lib/matplotlib/ticker.py

```diff
@@ -2079,13 +2254,13 @@ def decade_up(x, base=10):
     return base ** lx
 
 
-def is_decade(x, base=10):
+def is_decade(x, base=10, *, rtol=1e-10):
```

Comment:
Should this be `atol` instead of `rtol` ?

Reply:
It is a absolute tolerance after log transform, but a relative tolerance before log transform, therefore, for me both are correct, so I follow @anntzer choice.